modules/polling/api/fetchPolls.ts

```diff
@@ -68,26 +68,30 @@ export async function fetchAllPollsMetadata(pollList: PollSpock[]): Promise<Poll
 
 export async function fetchSpockPolls(
   network: SupportedNetworks,
-  queryFilter?: PollsQueryVariables | null
+  queryVariables?: PollsQueryVariables | null
 ): Promise<PollSpock[]> {
   const requestData = {
     chainId: networkNameToChainId(network),
     query: allWhitelistedPolls,
-    variables: queryFilter
+    variables: queryVariables
   };
 
   const data = await gqlRequest<GqlQuery>(requestData);
 
-  const pollList: Maybe<ActivePollsRecord>[] = data.activePolls.nodes;
-  //TODO: maybe we should use the GQL generated types when applicable
+  const pollEdges: ActivePollEdge[] = data.activePolls.edges;
+
+  const pollList = pollEdges.map(({ node: poll, cursor }) => {
+    return { ...poll, cursor };
```

Comment:
i don't see where this cursor is used

Reply:
we can't do serverside pagination right now because some of the filters are done in the client side. Soon I hope we can look into adding the categories to spock and then we can do filtering and pagination in the gql request. I left the cursor in there so it's available to use. We can still do SS pagination on the main `/polling` page.